src/index.test.ts

```diff
@@ -50,5 +52,6 @@ describe("exports", () => {
     expect(packageExports.revokeVerifiableCredential).toBe(
       revokeVerifiableCredential
     );
+    expect(packageExports.isValidVc).toBe(isValidVc);
```

Comment:
Interesting! in other parts of our codebase we just do with testing the package exports are defined, and not what they're defined as..